### DISCUSSION TITLE
How to configure immich to use in an association?

### DISCUSSION THREAD

**Discussion by MrCoala** [immich-app/immich#7096] [Q&A] [answered]
Hey,

i just fired up another immich instance for the use in an small private association, we are friends and we want to work on the same pictures. So no one should have separate pictures, everyone should be able to maintain all available pictures and upload new ones. Is it possible to configure it that way?

**Comment by alextran1502** [comment, marked as answer]
Use the same account might work

**Reply by MrCoala** [reply]
Thanks for the quick reply. Will stick with your suggestion, why make it complicated when it can be simple!
